src/schoology-extractor/schoology_extractor/helpers/export_data.py

```diff
@@ -44,5 +46,7 @@ def to_string(data: list) -> str:
     """
     assert isinstance(data, list)
 
+    logging.info("Converting data to string")
     df = pd.DataFrame(data)
+    logging.info("The data has been converted to string")
```

Comment:
Is it really important for the user at the command line to know about these two messages?

Reply:
Specifically this, probably not. The one of csv generated, I'd say yes... This shows the result direct in the console, the to_csv does not do it by itself.

Reply:
Just seems weird to me to have info messages on a `to_string` method. Doesn't seem important enough to write a log message. If keeping it, perhaps change to a Debug message?